students/TinaB/lesson05/mailroom_part3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+"""
+$ chmod +x mailroom_part3.py
+"""
+import datetime
+import os
+
+DONORS_DICT = {"William Gates": [326892.24, 122, 22, 12],
+               "Mark Zuckerberg": [30, 60, 65982.55],
+               "Jeff Bezos": [52636.27],
+               "Paul Allen": [877.33, 22],
+               "Steven Hawking": [326892.24, 123, 123.33, 123, 123],
+               "Justin Timberlake": [999658.25, 1233, 123]}
+
+# Menus
+
+
+def menu_selection(prompt, dispatch_dict):
+    """menu function"""
+    try:
+        while True:  # loop forever until they quit
```

Comment:
The reason your program does not quit is because you have two while True loops, one above here and another on line 216.

Reply:
Thanks for finding that. I discovered it this weekend - and honestly could've kicked myself for the stupidity of that line! I was so mad at myself for one - writing it and two not catching it right away!

Reply:
it happens to all of us 🙂